docs/try-flink/table_api.md

```diff
@@ -123,11 +123,14 @@ tEnv.registerTableSource("transactions", new BoundedTransactionTableSource());
 tEnv.registerTableSink("spend_report", new SpendReportTableSink());
 tEnv.registerFunction("truncateDateToHour", new TruncateDateToHour());
 
-tEnv
-    .scan("transactions")
-    .insertInto("spend_report");
-
-env.execute("Spend Report");
+// build table program
+Table table = tEnv.scan("transactions");
+
+// trigger execution
+TableResult tableResult = table.executeInsert("spend_report");
```

Comment:
```
		TableResult tableResult = tEnv
				.scan("transactions")
				.executeInsert("spend_report");
```

Keep the same with archetype code? I also prefer this one because it's more concise for example.

Reply:
There are `The Query` step and `Execute` step in `Breaking Down The Code` section. This design is for these steps.